plugins/modules/cloudtrail_info.py

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/python
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: cloudtrail_info
+version_added: 5.0.0
+short_description: Gather information about cloudtrail in AWS
+description:
+  - Gather information about cloudtrail in AWS.
```

Comment:
I would write it **C**loudtrail because it's a product name.

Reply:
technically it's **C**loud**T**rail.  (https://docs.aws.amazon.com/awscloudtrail/latest/userguide/how-cloudtrail-works.html) with the resources themselves being referred to as "trails" (lower case).